docs/REST API/rest-api.md

```diff
@@ -3536,6 +3592,128 @@ Retrieves a list of the configuration profiles assigned to a host.
 }
 ```
 
+### Get host's past activity
+
+`GET /api/v1/fleet/hosts/:id/activites/past`
+
+#### Parameters
+
+| Name | Type    | In   | Description                  |
+| ---- | ------- | ---- | ---------------------------- |
+| id   | integer | path | **Required**. The host's id. |
+| page | integer | query | Page number of the results to fetch.|
+| per_page | integer | query | Results per page.|
+
+#### Example
+
+`GET /api/v1/fleet/hosts/12/activites/past`
+
+##### Default response
+
+`Status: 200`
+
+```json
+{
+  "activities": [
+    {
+      "created_at": "2023-07-27T14:35:08Z",
+      "id": 2,
+      "actor_full_name": "Anna",
+      "actor_id": 1,
+      "actor_gravatar": "",
+      "actor_email": "anna@example.com",
+      "type": "ran_script",
+      "details": {
+        "host_id": 1,
+        "host_display_name": "Steve's MacBook Pro",
+        "script_name": "set-timezones.sh",
```

Comment:
@noahtalerman  I see we added the `script_name` field to the "ran_script" activity, and the `async` field is absent, is this deliberate? This is what the activity currently looks like: https://fleetdm.com/docs/using-fleet/audit-logs#ran-script

Reply:
Hopped on a call to discuss. We decided to keep `async` because removing it would be a breaking change.

We to add `script_name` so that we can show this name to users in the UI.